packages/types/src/project-config.ts

```diff
@@ -66,18 +66,7 @@ export interface ProjectDockerConfig {
 }
 
 /** Supported programming languages that each project can be written in. */
-export type LanguageType =
-	| 'bash'
-	| 'batch'
-	| 'go'
-	| 'javascript'
-	| 'php'
-	| 'python'
-	| 'ruby'
-	| 'rust'
-	| 'typescript'
-	| 'unknown'
-	| string;
+export type LanguageType = 'bash' | 'batch' | 'go' | 'javascript' | 'php' | 'python' | 'ruby' | 'rust' | 'typescript' | 'unknown' | string;
```

Comment:
Pretty sure all these changes will fail prettier.

The `just schemas` command will run prettier for you, but you probably need to install node modules with `yarn install`.